### PR TITLE
applied black formatting (v19.10b0)

### PR DESCRIPTION
**Pull request by jvanasco**
IIRC, @zzzeek integrated ``black`` as part of SqlAlchemy's code review via commit hooks.  I'm doing some housekeeping, and figured I'd simply do a PR to reformat this library with black.

Mike, if it's easier for your workflow to just apply ``black`` on your master and push, that works too.

### CODE REVIEW

**Comment by zzzeek** [comment]
OK....we did this, I thought, in f88e9d311f5562b0cbf793dd4e84 and the pre commit hook was even added in da31f6a4 so....did we do it wrong?

**Comment by zzzeek** [comment]
oh did you just re-run it without -l 79?  i think that's what I'm seeing.  old school only here l79 !

**Comment by zzzeek** [comment]
also the pre commit hook has r19.3b0.   so we can upgrade ..which means we should upgrade it on the other projects too

**Comment by zzzeek** [comment]
so do this - apply the new black version to the .pre-commit-config.yaml file , then do pre-commit run which will get it w/ all the settings.  also i am putting flake8 into the precommit file now to reduce pushes of dumb pep8 errors, see https://github.com/sqlalchemy/sqlalchemy/blob/master/.pre-commit-config.yaml

**Comment by jvanasco** [comment]
ok. this was a mix of changes in `black` and the line size. I only picked up 2 reformattings otherwise.

i can update the pre-commit yaml

what about adding a `pyproject.toml` file with the `black` settings and migrating the commandline args to that?  That's the only file `black` will consult (https://black.readthedocs.io/en/stable/pyproject_toml.html) and what `black` recommends as the best practice (https://black.readthedocs.io/en/stable/version_control_integration.html)

since i have your attention in this conversation:

* should I do another ticket/PR that updates the readme with a link to SqlAlchemy's Code of Conduct?
* is there anything we can do to the Readme or docs in SqlAlchemy/Dogpile to state "contributors: please install precommit and black" ?

**Comment by zzzeek** [comment]
there's some reason I had that I didn't want to do pyprojecvt.toml's yet, I think because there are still some tools that dont use them, some that do, and things get all mixed up.  But i can't remember which tools aren't using it, I have a vague recollection that one of them "never" will which is a bit spooky.   looks like pytest and flake8 are both on board but they are TODOs.   my own zimports would need it also.

I'd prefer to get rid of setup.cfg entirely before going to pyproject.toml and that means everyone should be on board with it.  but if people's editors  have "black" plugins that aren't working, then I guess we'd have no choice to have multiple config files to suit everyone's "best practice".

code of conduct: sure, I would need this in alembic /mako also , the goal is to have as much consistency as possible across all of these but things if possible...

instructions to install pre-commit:  well I guess in theory all the sqlalhcemy projects would point people to https://www.sqlalchemy.org/develop.html#style where we should have updated content for all this stuff

**Comment by jvanasco** [comment]
MIke - if the following is okay with you, just close this PR and I'll do the following

1. I'm going to make a handful of tickets to handle the code-of-conduct & pointers to develop for the various projects, then assign them to myself for PRs.

2. I'll make a ticket for `pyproject.toml` highlighting the migration concern for black and pros/cons in sqlalchemy, and stub it with info from here. i'll push a note to the google group so we get insight from other contributors.  once that is all hashed out, i'll make tickets and PRs in the other projects.

3. I'll make a sqlalchemy ticket to discuss updates to the `/develop` page on sqlachemyorg and handle generating a PR.

i'm 100% open to other methods, this just seems like the easiest way to get this stuff done but offload as much of it from you as possible.

**Comment by zzzeek** [comment]
> MIke - if the following is okay with you, just close this PR and I'll do the following
> 
>     1. I'm going to make a handful of tickets to handle the code-of-conduct & pointers to develop for the various projects, then assign them to myself for PRs.

Just make the PRs, there's no need for more tickets.   github PRs are more or less issues anyway.


> 
>     2. I'll make a ticket for `pyproject.toml` highlighting the migration concern for black and pros/cons in sqlalchemy, and stub it with info from here. i'll push a note to the google group so we get insight from other contributors.  once that is all hashed out, i'll make tickets and PRs in the other projects.

OK for that part, I want to look into what it will take for us to drop setup.cfg.    I don't think it's possible yet, but if you can identify any tools in our setup that are hostile to pyproject.toml that would help as I have a vague recollection one project or another was.     but for the black part at least, we might as well add the file.

> 
>     3. I'll make a sqlalchemy ticket to discuss updates to the `/develop` page on sqlachemyorg and handle generating a PR.

Ok if we need discussion then we can make a ticket for that.

> 
> 
> i'm 100% open to other methods, this just seems like the easiest way to get this stuff done but offload as much of it from you as possible.

I just waited in the DMV for four hours so my brain is total mush right now, I'll leave you to it thanks for the help :)

File: docs/build/changelog.rst [hunk 1]
```diff
@@ -6,6 +6,16 @@ Changelog
     :version: 0.9.1
     :include_notes_from: unreleased
 
+    .. change::
+        :tags: feature
+
```
Comment (zzzeek) [inline docs/build/changelog.rst]: just an FYI in order for me to avoid merge conflicts, place a file-per-change in docs/build/unreleased

**Comment by zzzeek** [comment]
I've upgraded black and added pyproject.toml in https://gerrit.sqlalchemy.org/#/c/sqlalchemy/dogpile.cache/+/1670/ which is not merged yet, if you want to review.

**Comment by jvanasco** [comment]
> I've upgraded black and added pyproject.toml in https://gerrit.sqlalchemy.org/#/c/sqlalchemy/dogpile.cache/+/1670/ which is not merged yet, if you want to review.

thanks, you saved me doing the same change.

the only thing I would have done differently...

since black now standardizes the quotes for `__version__`  in `__init__.py`, setup.py no longer needs the `['"]` Regex pattern.  

    -    re.compile(r""".*__version__ = ["'](.*?)["']""", re.S)
    +    re.compile(r'''.*__version__ = "(.*?)"''', re.S)

**Comment by zzzeek** [comment]
until black changes again....so...I don't mind leaving it :)

**Pull request closed without merging**